content/docs/other-guides/support.md

```diff
@@ -0,0 +1,168 @@
++++
+title = "Support"
+description = "Where to go with questions and suggestions"
+weight = 110
++++
+
+This page describes the Kubeflow resources and support options that you can
+explore when you encounter a problem, have a question, or want to make a
+suggestion about Kubeflow.
+
+TODO: This page currently mentions an "alpha" status whereas the version policy
+uses "experimental". I'm proposing a change to "alpha". If we decide to keep
+"experimental", I'll change this page.
+
+## Levels of support
+
+Kubeflow applications offer various levels of support, based on the application
+status. To see the status of each application, refer to the 
+[Kubeflow application 
+matrix](/docs/reference/version-policy/#kubeflow-application-matrix) on the
+version policies page.
+
+The following table describes the level of support that you can expect based on
+the status of an application:
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Application status</th>
+        <th>Level of support</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Stable</td>
+        <td>TODO What are the expectations here? Some examples:
```

Comment:
Noting the TODO.

@jbottum and @abhi-g and @jlewi Can you help fill in the support expectations for **stable** apps?

Reply:
Done.